src/App.tsx

```diff
@@ -1,8 +1,72 @@
-import React from 'react';
+import React, { useState } from 'react';
 import './App.css';
 
-const App = () => (
-  <h1>Dynamic list of TODOs</h1>
-);
+import { getPrepareTodos } from './helpers/api';
+import TodoList from './components/TodoList';
+
+const App: React.FC = () => {
+  const [todos, setTodos] = useState<ApdateTodo[]>([]);
+  const [initialTodos, setInitialTodos] = useState<ApdateTodo[]>([]);
```

Comment:
on todos list in the state is enough, the sorted list you can get just before the render